packages/sidenav/src/SidenavItem.ts

```diff
@@ -44,6 +44,9 @@ export class SideNavItem extends LikeAnchor(Focusable) {
     @property({ type: Boolean, reflect: true })
     public expanded = false;
 
+    @property({ reflect: true })
+    public role = 'listitem';
```

Comment:
Can you see a path towards this ever _not_ being "listitem"? If not, we should probably cover over it by not making it a public property. If so, should we document cases where a user would override this?

Reply:
It should probably be private, but if it is, can it still be reflected?

Reply:
In `firstUpdated` we can do `this.setAttribute(‘role', ‘listitem’)`, if there _shouldn’t_ be user overrides.

Reply:
That's what I've done.